lib/rules/avoid-inline-spacing.json

```diff
@@ -0,0 +1,12 @@
+{
+	"id": "avoid-inline-spacing",
+	"selector": "[style]",
+	"tags": ["wcag21", "wcag1412"],
+	"metadata": {
+		"description": "Ensure text spacing is not affected by inline spacing styles that affects CSS specificity",
+		"help": "No inline styles with '!important' that affect text spacing should been specified"
```

Comment:
I'm not sure about this "should be specified". I don't think these texts describe the problem well. The point is not to prevent custom stylesheets from overriding whitespace, and the reason that I want to include this "not supported in many browsers" thing somewhere is because this is an accessibility support problem. Our messaging needs to make it clear that this is a support problem, and not something thats inherently inaccessible.